libs/components/src/lib/functional-header/stories/vl-functional-header.stories.ts

```diff
@@ -4,12 +4,16 @@ import { Meta, StoryFn } from '@storybook/web-components';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import functionalHeaderDoc from './vl-functional-header.stories-doc.mdx';
 import { functionalHeaderArgs, functionalHeaderArgTypes } from './vl-functional-header.stories-arg';
+import { filterOutClasses } from '@domg-wc/common-utilities';
 
 export default {
     title: 'Components/functional-header',
     argTypes: functionalHeaderArgTypes,
     parameters: {
-        docs: { page: functionalHeaderDoc },
+        docs: {
+            page: functionalHeaderDoc,
+            transformSource: filterOutClasses,
```

Comment:
had deze (`filterOutClasses`) nog niet gezien (maar herinner me dat je er van verteld had) - goede toevoeging!

zouden we deze mss in `apps/storybook/.storybook/preview.js` zetten? of beter voorzichtig enkel per story toepassen?